Vanity link clarification plus some trivial cosmetic fixes on the Keyboard Shortcuts page (#186)
Microsoft-community/Community-website: src/mod/docs/index.md

Automatic actions performed by Cliptok will still trigger on messages that were blocked by Automod (see below).
:::

> ::: tip Server Invites
> Action will only be taken when there are unapproved invites. Certain approved invite links can be sent by any member regardless of their role (provided that they are not muted)
> :::

Reviewer: ```suggestion
::: tip Note
Action will only be taken when unapproved invites are being sent. Certain approved invite links can be sent by any member regardless of their role, provided that they are not muted.
:::
```

Automatic actions performed by Cliptok will still trigger on messages that were blocked by Automod (see below).
:::

::: tip Server Invites
Action will only be taken when there are unapproved invites. Certain approved invite links can be sent by any member regardless of their role (provided that they are not muted)
:::

> ::: details Approved vanity links

Reviewer: ```suggestion
::: details Approved Vanity Links
```
